sympy/printing/llvmjitcode.py

```diff
@@ -0,0 +1,192 @@
+from __future__ import print_function, division
+
+'''
+Use llvmlite to create executable functions from Sympy expressions
+
+This module requires llvmlite (https://github.com/numba/llvmlite).
+'''
+
+import ctypes
+
+from sympy.external import import_module
+from sympy.printing.printer import Printer
+from sympy import S
+from sympy.utilities.decorator import doctest_depends_on
+
+llvmlite = import_module('llvmlite')
+if llvmlite:
+    ll = import_module('llvmlite.ir').ir
+    llvm = import_module('llvmlite.binding').binding
+    llvm.initialize()
+    llvm.initialize_native_target()
+    llvm.initialize_native_asmprinter()
+
+
+class LLVMJitPrinter(Printer):
```

Comment:
The LLVM IR representation can be printed (it looks a bit like assembly code).  It might be useful to have a function that returns the IR.  What I don't know yet is if fragments of LLVM IR can be produced, or if the entire function needs to be produced at once.  The current code printers produce single C or Fortran expressions, and the codgen code puts them in functions.  I don't know if the split is possible or reasonable for creating LLVM IR.